src/components/Zodiac/zodiac.css

```diff
@@ -0,0 +1,7 @@
+.container {
+    background-color: whitesmoke;
+    width: 150px;
+    height: 300px;
+    padding: 5px;
+    color: #ff0000;
+  }
```

Comment:
Excellent work Sydnie! Your app looks amazing! Here we can add opacity: 50% to give the background-color some transparency.